### DISCUSSION TITLE
how do i flixel handler crash like indie cross? (an fnf mod)

### DISCUSSION THREAD

**Discussion by vyroxhub** [HaxeFlixel/flixel#2561] [Q&A] [answered]
read the title

**Comment by RichardBray** [comment, marked as answer]
I believe for handling crashes in flixel there's a tool called crash dumper by @larsiusprime 

https://github.com/larsiusprime/crashdumper

I've never used it before so I'm not sure how it works 🤷

**Reply by vyroxhub** [reply]
ok

**Comment by ghost** [comment]
This isn't the place for asking FNF questions silly.

**Reply by vyroxhub** [reply]
haha good joke, have a good day
_edited by GeoKureli_

**Reply by vyroxhub** [reply]
also sorry for being rude

**Reply by Geokureli** [reply]
Don't be rude and then apologize, just edit it to be not rude. You asked a fair question, and I think hayfii is just messing with you

**Reply by vyroxhub** [reply]
oh
